.github/workflows/docs.yml

```diff
@@ -15,10 +15,16 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      # NOTE: Comment out when https://github.com/rust-lang/mdBook/pull/1306 is merged and released
+      # - name: Setup mdBook
+      #   uses: peaceiris/actions-mdbook@v1
+      #   with:
+      #     mdbook-version: "0.4.10"
+
+      # NOTE: Delete when the previous one is enabled
       - name: Setup mdBook
-        uses: peaceiris/actions-mdbook@v1
-        with:
-          mdbook-version: "0.4.10"
+        run: |
+          cargo install --git https://github.com/Ruin0x11/mdBook.git --branch localization --locked mdbook
```

Comment:
If I understand correctly, I don't think --locked does anything here because the build data isn't persistent, so the cargo.lock is regenerated every time. We could use the [--rev](https://doc.rust-lang.org/cargo/commands/cargo-install.html#install-options) flag to lock to a specific commit.

Reply:
I added the `locked` part because it was complaining that the repo has 2 binaries and did not know which one to install. `locked` solved that, and also tried with `-p` but it's not supported by `cargo install` it seams. Maybe there is another way more correct to achieve the same, but don't know about it

Reply:
Nevermind, saw my mistake and added the `--rev` part